test/middleware/logging.test.ts

```diff
@@ -1,25 +1,31 @@
-const request = require('supertest')
-const express = require('express')
-const {logger} = require('../../src/logger')
-const {logRequest} = require('../../src/middleware/logging')
+import express from 'express'
+import request from 'supertest'
+import { logger } from '../../src/logger'
+import { logRequest } from '../../src/middleware/logging'
 
 describe('logging', () => {
-  let server, output
+  let server: express.Express
+  let output: any[]
 
   beforeAll(() => {
-    logger.addStream({
+    const stream: any = {
```

Comment:
I set this type to `any` since we are creating a mock stream to add to the test logger instance.